kubernetes/ansible/roles/mount-keys/tasks/encrypt.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
why do we need  this ?

Reply:
Its yaml identifier right.

Reply:
Not that... The play book ... Encrypt again?

Reply:
Its used to encrypt the private keys after generating public keys from the decrypted private keys.

Reply:
so we don't need this anymore right

Reply:
Another option would be to just copy the data from private back to inventory directory which would be encrypted. Or just remove the keys directory post install from inventory directory that way we can save time not to encrypt again. In case of debugging, we would need to copy the keys directory manually again and decrypt if we remove the directory.

Reply:
Please let me know what I should change to. Do we copy keys directory again just for the sake of original inventory copied structure or remove the directory

Reply:
Changed to delete model